test/mocha.opts

```diff
@@ -1,2 +1,2 @@
---compilers coffee:coffee-script/register
+--require coffee-script/register
```

Comment:
I noticed mocha has deprecated `mocha.opts` --- would you delete this file and move the config into `package.json`?  This snippet seems equivalent:
```
  "mocha": {
    "require": "coffee-script/register",
    "reporter": "spec"
  }
```

Reply:
Your suggestion seems to have worked beatifully. Thanks.